RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/TouchHelper.java

```diff
@@ -415,6 +431,22 @@ public void run() {
             recyclerView.invalidate();
         }
 
+        private void animateDismissItem(View selectedItemView, float translationX) {
+            final TranslateAnimation translateAnimation =
+                    new TranslateAnimation(translationX, 0 - selectedItemView.getWidth(), 0, 0);
+            translateAnimation.setDuration(150);
+            translateAnimation.setAnimationListener(new DismissAnimationListener(TouchHelper.this.selected));
+            selectedItemView.startAnimation(translateAnimation);
```

Comment:
`selectedItemView#setHasTransientState(true)` to prevent improper recycle of item view (you also need to call `setHasTransientState(false)` on the end of animation).